http2_general/test_flow_control_window.py

```diff
@@ -52,15 +67,28 @@ def test_single_stream(self):
         6. Client send WindowUpdate with 1K flow control window after receiving first DATA frame;
         7. Tempesta forward second DATA frame with 1k bytes;
         """
```

Comment:
Please add comment here. Why should we check the flow control for a response with a trailer? The flow control works with body, so I don't understand the effect of trailers.

Reply:
Yes, but I found error in appropriate @kingluo PR using this test.